tensorflow/python/ops/rnn_cell.py

```diff
@@ -85,9 +85,11 @@ def _state_size_with_prefix(state_size, prefix=None):
 
 
 class RNNCell(object):
-  """Abstract object representing an RNN cell.
+  """Array of RNN cells.
```

Comment:
sounds like you want to change the notation of cell to mean something else here.  however, we already have a meaning in this code for "Cell" and it's:

An RNN cell, in the most abstract setting, is anything that has a state and performs some operation that takes a matrix of inputs.

So - don't change all the names of RNNCell objects to mean "cell" as something else.  An RNNCell creates one unit of computation, it is not an "Array of RNN cells".

Rather, add a paragraph explaining what the word "cell" means in the literature and how that differs from what we mean by Cell.  That's it.